vanilla-framework/vanilla-framework
# Corrects colorpicker and datetime errors

## Done

- Corrects `datatime` typo to `datetime`
- Stops styling color pickers as if they’re text fields, since browsers make them behave as buttons.

## QA

Observe improvement in the appearance of:
- `<input type="datetime">` — looking more consistent with other Vanilla fields
- `<input type="color">` — looking more like color picker buttons usually do.

## Details

I discovered these two bugs when researching #319.

Eventually, perhaps, color pickers should be styled to look more like Vanilla buttons. But:
- this could be tricky for something without an internal label, since Vanilla buttons don’t look much like buttons
- this would be low priority, since if anyone had been using Vanilla color pickers they would have noticed the mistaken styling by now.

## Screenshots

Before:
![vanilla-color-before](https://user-images.githubusercontent.com/19801137/48977654-5bc1fa80-f096-11e8-88df-584966cac655.png)

After:
![vanilla-color-after](https://user-images.githubusercontent.com/19801137/48977653-5bc1fa80-f096-11e8-8ffa-9518b3f6b37a.png)
